### PR TITLE
feat: CreateUser function

### PR DESCRIPTION
**Pull request by yonatandt**
Signed-off-by: YonatanTal <yonatald@gmail.com>

### CODE REVIEW

File: createUser/function.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+  "bindings": [
+    {
+      "name": "createUserMessage",
+      "type": "queueTrigger",
+      "direction": "in",
+      "queueName": "user-queue",
+      "connection": "radarstrg_STORAGE"
+    }
+  ],
+  "scriptFile": "../dist/createUser/index.js"
```
Comment (carmeltsadok) [inline createUser/function.json:11]:
```suggestion
  "scriptFile": "../dist/CreateUser/index.js"
```
Change folder's name as well. (To do so, commit twice- one without the folder, and one with the uppercase name)

File: createUser/readme.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# QueueTrigger - TypeScript
```
Comment (carmeltsadok) [inline createUser/readme.md:1]: Remove this file

**Review by carmeltsadok — CHANGES_REQUESTED** [review]

**Review by carmeltsadok — APPROVED** [review]

File: CreateUser/index.ts [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+import { User } from '../shared/user/user.interface';
+import UserModel from '../shared/user/user.model';
+import getConnection from '../shared/mongoose';
+
+const createUser: AzureFunction = async (context: Context, createUserMessage: User): Promise<void> => {
+    try {
+        await getConnection();
+        await UserModel.create(createUserMessage);
+    } catch (err) {
+        context.log.error(err + context.invocationId);
```
Comment (lustigy96) [inline CreateUser/index.ts:11]: why do we need this context.log ? 
Don't think it suppose to be in the dev version

**Review by lustigy96 — CHANGES_REQUESTED** [review]

File: CreateUser/index.ts (hunk 3 above)
Comment (yonatandt) [reply CreateUser/index.ts:11]: We still did not decide on how to handle our logs, so in the meantime, I just use `contet.log`.

**Review by lustigy96 — APPROVED** [review]

**Pull request merged**